libopenage/build.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+#include "build.h"
+
+#include <libopenage/config.h>
+
+
+namespace openage {
+
+
+build_configuration::build_configuration()
+	:
+	global_asset_dir{config::global_asset_dir},
+	config_options{config::config_option_string},
+	build_src_dir{config::buildsystem_sourcefile_dir},
+	compiler{config::compiler},
+	compiler_flags{config::compiler_flags},
+	cython_version{config::cython_version} {
+
+}
+
+
+build_configuration get_build_configuration() {
+	build_configuration config;
```

Comment:
This must be static, as it's some compiletime stuff. We could just place all this as a struct in `config.cpp`,
so the things you copy from `config::` could be just a static struct in `config::`.

the `get_build_configuration` can then simply return that struct for py or whatever.